CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing
+
+We would ❤️ for you to contribute to Appwrite and help make it better! As a contributor, here are the guidelines we would like you to follow:
+
+## Code of Conduct
+
+Help us keep Appwrite open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+## Getting Started
+
+Install using composer:
+```bash
+composer require appwrite/php-runtimes
+```
+
+```php
+<?php
+
+require_once '../vendor/autoload.php';
+
+use Appwrite\Runtimes\Runtimes;
+
+$runtimes = new Runtime();
+
+// returns all supported cloud function runtimes
+Runtimes::getAll();
+```
+
+## Adding a new Runtime
+
+After the Docker image is created, this must be added to the main class of this library.
+
+References to this must be added to the constructor of `src/Runtimes/Runtimes.php`.
+
+Example:
+
+```php
+$dotnet = new Runtime('dotnet', '.NET');
+$dotnet->addVersion('5.0', 'mcr.microsoft.com/dotnet/runtime:5.0-alpine', 'appwrite/env-dotnet-5.0:1.0.0', [System::X86, System::ARM]);
+$runtimes[] = $dotnet;
+```
+
+## Function Runtimes Checklist
+
+The following checklist aims to ensure that a function runtime gets added successfully
+
+- [ ] Implement the runtime in [open-runtimes/open-runtimes](https://github.com/open-runtimes/open-runtimes) (you can find the tutorial [here](https://github.com/open-runtimes/open-runtimes/blob/main/docs/add-runtime.md))
+  - [ ] Prepare the files for your new runtime
+    - [ ] Dockerfile
+    - [ ] Readme
+  - [ ] Write the runtime
+    - [ ] Initialize a web server
+      - [ ] Set Port 3000
+      - [ ] Bind IP 0.0.0.0
+      - On each POST Request
+        - [ ] Check that the `x-internal-challenge header` matches the `INTERNAL_RUNTIME_KEY` environment variable
+        - [ ] Decode the executor's JSON POST request
+    - [ ] Create Request Class
+      - [ ] Fields
+        - [ ] env
+        - [ ] payload
+        - [ ] headers
+    - [ ] Create Response Class
+      - [ ] Functions
+        - [ ] send(string)
+        - [ ] json(object)
+    - [ ] Execute the function
+      - [ ] Add `try catch` block for error handling
```

Comment:
Try to provide as many human-readable errors as possible. Like check if file or method is missing.

Reply:
Are there any specific errors we're looking to add aside from the two mentioned here @Meldiron?

Reply:
Should be language specific. These two should probably be checked always unless that check is done during build process. Cant think of anything else.

Reply:
```suggestion
      - [ ] Add `try catch` block for error handling
        - [ ] Check if the file is missing
        - [ ] Check if the method is missing
        - [ ] Any other language-specific errors that may occur 
```

@Meldiron how does this look?

Reply:
Amazing.

Reply:
Awesome... committing these 🙌